packages/manager/modules/hub/src/components/user-panel/user-infos/template.html

```diff
@@ -0,0 +1,74 @@
+<div class="manager-hub-user-infos">
+    <span
+        class="manager-hub-user-infos_initials"
+        data-ng-class="{
+            'manager-hub-user-infos_initials-centered': !$ctrl.isExpanded,
+            'float-left': $ctrl.isExpanded
+          }"
+    >
+        <span data-ng-if="$ctrl.me" data-ng-bind="$ctrl.getNameInitials()">
+        </span>
+    </span>
+    <p class="oui-chip" data-ng-if="!$ctrl.isExpanded">
+        <oui-skeleton
+            class="mx-auto"
+            data-ng-if="!$ctrl.supportLevel"
+            data-size="xs"
+        ></oui-skeleton>
+        <small
+            data-ng-if="$ctrl.supportLevel"
+            data-ng-bind="('hub_user_support_level_' + $ctrl.supportLevel.level) | translate"
+        ></small>
+    </p>
+    <p data-ng-if="!$ctrl.me">
+        <oui-skeleton class="mx-auto" data-size="s"></oui-skeleton>
+    </p>
+    <div
+        class="manager-hub-user-infos_user-name mb-2"
+        data-ng-class="{
+         'manager-hub-user-infos_user-name__expanded': $ctrl.isExpanded,
+       }"
+        data-ng-if="$ctrl.me"
```

Comment:
Can the `ng-if="$ctrl.me"` be grouped ? It might help to understand what is diplayed if $ctrl.me is defined